OpenQA/Isotovideo/Utils.pm

```diff
@@ -128,7 +128,6 @@ configuration variables.
 sub handle_generated_assets ($command_handler, $clean_shutdown) {
     my $return_code = 0;
     # mark hard disks for upload if test finished
-    return unless $bmwqemu::vars{BACKEND} eq 'qemu';
```

Comment:
`if backend::foo->can('do_extract_assets')` should be enough to check, no extra method needed

Reply:
Shouldn't it be enough to not set `PUBLISH_HDD_1` and friends in the settings?

Reply:
@perlpunk That's also a good idea but requires to get rid of `sub do_extract_assets ($self, $args) { $self->notimplemented }` in `baseclass.pm` of course.

Reply:
oh right. stupid me.
It would need to be `if defined &backend::svirt::do_extract_assets`
This would also need to be fixed in https://github.com/os-autoinst/os-autoinst/pull/1895
Or we really get rid if the baseclass implementation, right.

Reply:
I actually like the idea because

> Or we really get rid if the baseclass implementation, right.

doesn't seem too intrusive to me. I doubt that so far some code has tried to invoke `do_extract_assets` looking for the "not implemented" exception.

Reply:
I have `defined &backend::svirt::do_extract_assets`, so no need to remove the "notimplemented" function - I assume we'd rather keep it if we can do that it can raise an error in other backends.

Reply:
Using `defined` should do it for now.
But `can()` would be more correct because it uses inheritance. Two backend classes could share the same code.
So removing the "notimplemented" methods, doing a `can()` check instead and having an informative message would save the same purpose as the "notimplemented" methods.